action.yml

```diff
@@ -32,7 +32,7 @@ runs:
       run: |
         curl -L --silent --show-error --fail "https://download.panic.com/playdate_sdk/PlaydateSDK-${{ inputs.version }}.zip" -o sdk.zip
         unzip sdk.zip
-        sudo installer -store -pkg "PlaydateSDK.pkg" -target /
+        sudo installer -store -pkg "PlaydateSDK-${{ inputs.version }}.pkg" -target /
```

Comment:
Instead of using version, please add `mv ./*.pkg ./PlaydateSDK.pkg` after unzip (line 34). I think I would better and prevents future beaks when Panics rename pkg again :)

Reply:
We're using [similar incredible technique for linux](https://github.com/pd-rs/get-playdate-sdk/blob/main/action.yml#L55).